tests/integration/techRecords.intTest.ts

```diff
@@ -290,6 +294,237 @@ describe("techRecords", () => {
       });
     });
   });
-});
 
+  describe("postTechRecords", () => {
+    context("when database is populated", () => {
+      beforeAll(async () => {
+        await emptyDatabase();
+      });
+
+      afterAll(async () => {
+        await populateDatabase();
+      });
+
+      context("when database is populated", () => {
+        beforeEach(async () => {
+          await populateDatabase();
+        });
+
+        afterEach(async () => {
+          await emptyDatabase();
+        });
+
+        // @ts-ignore
```

Comment:
I don't like that this is needed. When I take it out, it complains because the data is *wrong*. I see that you didn't alter the data, so I won't ask you to fix it, but we shouldn't sweep stuff like this under the rug either.

Reply:
I think a possible cause to this is that the structure of the object has changed and the interface wasn't updated to meet the last version of the API spec?